### PR TITLE
feat(unique_toolkit): New package template placeholder added.

### PR DESCRIPTION
**Pull request by sheiksadique**

### CODE REVIEW

File: unique_toolkit/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Unique-AG
```
Comment (mfadler) [inline unique_toolkit/LICENSE:3]: Not 2024 ?

File: unique_toolkit/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Unique Tool Kit
+
+This package provides highlevel abstarctions and methods on top of `unique_sdk` to ease application development for the Unique Platform.
```
Comment (mfadler) [inline unique_toolkit/README.md:3]: typo in abstractions

File: unique_toolkit/pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,71 @@
+[tool.poetry]
+name = "unique_toolkit"
+version = "0.0.1"
+description = ""
+authors = [
+    "Unique Data Science <datascience@unique.ch>",
+]
```
Comment (mfadler) [inline unique_toolkit/pyproject.toml:7]: Is this a good idea? Might get spammed a lot. I mean people can open issues when there are encountering any problems.

File: unique_toolkit/unique_toolkit/chat/service.py [hunk 4]
```diff
@@ -0,0 +1,182 @@
+import unique_sdk
+from unique_sdk.utils.chat_history import load_history
+
+from unique_toolkit.performance.async_wrapper import to_async
+
+from .state import ChatState
+
+
+class ChatService:
```
Comment (mfadler) [inline unique_toolkit/unique_toolkit/chat/service.py:9]: Can you please include all my implementations for now, I will make a separate commit for them.

**Review by mfadler — COMMENTED** [review]
Library setup looks good to me. Please exclude all my implementations, I will create a separate commit for them.

File: unique_toolkit/pyproject.toml (hunk 3 above)
Comment (sheiksadique) [reply unique_toolkit/pyproject.toml:7]: I don't believe this email exists at the moment. 🤷

File: unique_toolkit/unique_toolkit/__init__.py [hunk 5]
```diff
@@ -0,0 +1,3 @@
+from .observability.log_config import load_logging_config  # noqa: F401
+from .sdk.init import init_sdk  # noqa: F401
+from .security.verify import verify_signature_and_construct_event  # noqa: F401
```
Comment (mfadler) [inline unique_toolkit/unique_toolkit/__init__.py:3]: I think these dependencies should be removed.

Comment (sheiksadique) [reply unique_toolkit/unique_toolkit/__init__.py:3]: Thankyou. Should add tests to the pipeline.

**Review by mfadler — APPROVED** [review]

**Pull request merged**